alacritty_terminal/src/term/mod.rs

```diff
@@ -1112,7 +1112,7 @@ impl<T> Term<T> {
 
     pub fn swap_alt(&mut self) {
         if self.alt {
-            let template = Cell { bg: self.cursor.template.bg, ..Cell::default() };
+            let template = self.cursor.template;
             self.grid.region_mut(..).each(|c| c.reset(&template));
```

Comment:
You inlined it every else, might as well do the same here, right? Or does this exceed the 100 column limit?

Reply:
Borrow checker.